docs/evm/ethereum-vs-flow-evm.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Differences between EVM on Flow and Ethereum
+sidebar_label: EVM on Flow vs. Ethereum
+sidebar_position: 14
+---
+
+# Differences between EVM on Flow and Ethereum
+
+## Introduction
+
+The Flow network uses [Cadence](https://cadence-lang.org/) as its main execution environment. Cadence offers a safer, more efficient, and more developer-friendly experience for building smart contracts and decentralized applications. However, supporting EVM could make it easier for existing applications and tools already deployed for the EVM ecosystem to onboard to the Flow network ([more](https://developers.flow.com/evm/about) on this topic).
+
+EVM on Flow is designed with these major goals in mind: 
+
+- Supporting EVM equivalency: Ensure that any tools and applications deployed to or run on Ethereum can also be deployed and run on Flow.
+- Minimizing breaking changes to the Cadence ecosystem, software and tools
+- Maximum composability across environments: Allowing atomic and smooth interaction between EVM and Cadence environments.
+
+Let's look at some aspects of design to gain a better understanding:
```

Comment:
Don't think we need this sentence. The colon gives the impression that there will be a list but the format of the rest of the doc and differing header sizes makes it a bit strange.

Reply:
removed